modules/button/react/README.md

```diff
@@ -27,7 +14,44 @@ or
 yarn add @workday/canvas-kit-react-button
 ```
 
-## Usage
+## Deprecation Warning
+
+> We are rolling out our next iteration of our buttons and will be deprecating the old ones(orange
+> primary, secondary, and delete) on (this time frame). The biggest change is with regards to colors
+> and styling, but the behavior should remain the same.
+>
+> We've added a deprecation warning to the old buttons as well as a console warn in the component
+> itself. The message should read `This is now deprecated, use Button component`.
```

Comment:
The message in the console says `deprecation: Button is deprecated: This is now deprecated, use Button component` which is not very helpful.

Reply:
I don't know how we could differentiate between the two but it would be nice

Reply:
this has been updated in the readme as well!